### DISCUSSION TITLE
[website] Be able to share an icon or an icon search

### DISCUSSION THREAD

**Discussion by astorije** [vmware-clarity/core#27] [Ideas]
On https://clarity.design/foundation/icons/shapes/, could the current search value be added to the query string (e.g. `/shapes/?search=angle`) and/or the currently expanded icon be added as the hash of the URL (e.g. `/shapes/#icon-angle-double`)?

It would help a lot when referring to icons in documents/comments, and to share icon suggestions between developers and designers for example.

I noticed each icon already has an id attribute, but opening e.g. https://clarity.design/foundation/icons/shapes/#login does not scroll to the right icon as those seem to be loaded after the page gets rendered.

![image](https://user-images.githubusercontent.com/113730/162231237-9bddd2b5-8022-422c-9ca0-ecb1aa1cccae.png)

**Comment by d-m-s** [comment]
I've also stumbled upon this ... would be a nice-to-have feature!
